src/Options.cpp

```diff
@@ -95,6 +112,10 @@ void InitOptions(int argc, const char *argv[], Options &options) {
                 "\t--no-update          Skip applying launcher updates (you must update manually)\n"
                 "\t--no-launch          Skip launching the game (you must launch the game manually)\n"
                 "\t--dev                Developer mode, same as --verbose --no-download --no-launch --no-update\n"
+                #if defined(__APPLE__)
+                "\t--bottle <bottle>        Name of the Bottle where the game is located in\n"
+                "\t--bottle-path <path>     Path to the Bottle where the game is located in\n"
```

Comment:
This won't work if both are specified at once right? If so please indicate that in the message.